### PR TITLE
Sendgrid fix

### PR DESCRIPTION
**Pull request by jzeiders**
Fixes sendgrid & we now have a Tasks system

### CODE REVIEW

File: tasks/sendgridsync.js [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+
+const sendgridSync = async () => {
+  //Lazy Load Models so you can change process.ENV dynamically
+  const models = require("../api/models");
+  sgClient.setApiKey(process.env.SENDGRID_API_KEY);
+
+  const hackerProfiles = await models.HackerProfile.findAll({});
+
+  // Applicants Odyssey List in Sendgrid
+  const HackersListID = "95f43e1d-83c2-4818-a7d7-19eb44c30ae4";
```
Comment (saresend) [inline tasks/sendgridsync.js]: Is it gonna be a problem that this is checked into source control?

**Comment by jzeiders** [comment]
@saresend stamp plz

File: tasks/sendgridsync.js (hunk 1 above)
Comment (jzeiders) [reply tasks/sendgridsync.js]: It's less than ideal but fine non-secure item

Comment (jzeiders) [reply tasks/sendgridsync.js]: If we ever have to change it, will move to an ENV. It's not a security risk just a magic string

Comment (jzeiders) [reply tasks/sendgridsync.js]: If we ever have to change it, will move to an ENV. It's not a security risk just a magic string

Comment (saresend) [reply tasks/sendgridsync.js]: Okay, yeah that's fine just wanted to make sure this wasn't supposed to be secret

File: tasks/sendgridsync.js [hunk 2]
```diff
@@ -0,0 +1,45 @@
...
+        first_name: hp.firstName,
+        last_name: hp.lastName,
+        email: hp.email,
+        custom_fields: {
+          e9_T: hp.status,
+          e3_T: hp.school,
+          e10_D: hp.submittedAt
+        }
+      };
+    });
```
Comment (saresend) [inline tasks/sendgridsync.js]: If you're pulling values from the sequelize model, I think it's better to use the `get(/* fieldname */)` function. The hackerProfile model wraps a lot of functionality, so I don't think you can use the dot syntax to get attributes unfortunately

Comment (jzeiders) [reply tasks/sendgridsync.js]: It works, is there a reason not to? I believe it maps to the getter

**Pull request merged**